src/test/shared/awsFiletypes.test.ts

```diff
@@ -32,15 +33,23 @@ describe('awsFiletypes', function () {
             'python3.7-plain-sam-app/template.yaml'
         )
         cfnUri = vscode.Uri.file(cfnFile)
+
+        const buildspecFile = workspaceUtils.tryGetAbsolutePath(
+            vscode.workspace.workspaceFolders?.[0],
+            'buildspec/buildspec.yml'
+        )
+        buildspecUri = vscode.Uri.file(buildspecFile)
     })
 
     after(async function () {
         testUtil.closeAllEditors()
     })
 
     it('emit telemetry when opened by user', async function () {
-        await globals.templateRegistry.addItemToRegistry(cfnUri!)
+        await globals.templateRegistry.cfn.addItemToRegistry(cfnUri!)
+        await globals.templateRegistry.cfn.addItemToRegistry(buildspecUri!)
```

Comment:
should it be `globals.templateRegistry.buildspec...` ?

Reply:
Yeah that should have been buildspec 🤔